pages/about.html

```diff
@@ -0,0 +1,126 @@
+                    So if you're active and engaged in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together
+                </p>
+                <p class="intro-txt-abt-1">
+                    Please contact us per Email for nay further questions about CC Global Summit 2023!
+                </p>
+                <a class="email intro-txt-abt-1" href="mailto:sagiecyber@gmail.com">sagiecyber@gmail.com</a>
+            </section>
+        </section>
+    </header>
+    <main>
+        <section class="main-abt">
+            <h2 class="title-abt-1">
+                Creative Commons Global Summit 2023 Logo
+            </h2>
+            <p class="intro-txt-abt-2">
+                The logo of CC Global Summit 2023 was decided through the logo competition from 8, June to 7, July. 
+                After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize. 
+                After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:
+            </p>
+            <a href="#" class="logo-abk">BLK-NG</a>
+        </section>
+        <section class="main-abt-2">
+            <h2 class="title-abt-1">See the past Black Ng Global Summits</h2>
+            <p class="intro-txt-abt-2">Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+            <section class="pass-celebrate">
+               <article class="p-c">
+                    <div>
+                        <h2>2015</h2>
+                        <p class="intro-txt-abt-2">BLK-NG Summit in Lagos</p>
+                    </div>
+               </article> 
+               <article class="p-c">
+                    <div>
+                        <h2>2020</h2>
+                        <p class="intro-txt-abt-2">BLK-NG Summit in Edo</p>
+                    </div>
+                </article> 
+            </section>
+        </section>
+    </main>
+    <footer>
+        <section id="sponsors-abt" class="mb">
+            <h2 class="program-title wht">Partner</h2>
+            <ul class="sponsors">
+                <li>mosilla</li>
+                <li><span>Goog<span class="bd-1">l</span>e</span></li>
+                <li>NAVER</li>
+                <li>daumkakao</li>
+                <li>airbnb</li>
+            </ul>
+        </section>
+        <section class="buttom-f-nav-abt">
+            <a class="logo-abt-f" href="#">BLK-NG</a>
+            <article class="short-f-note">
+                <h2>2023 Creative Commons Korea Some Rights Reserved.</h2>
+                <p>
+                    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Modi.
+                </p>
```

Comment:
- [ ] 👍 Thank you for your efforts! As mentioned earlier, please adjust the color for this element, as it currently doesn't match the design. Kindly update your code to use the correct color as per the design. Keep up the good work! 🌟💻🚀